src/components/adaptive-cards/InputNumber/InputNumber.jsx

```diff
@@ -16,8 +17,30 @@ import InputField from '../../generic/InputField/InputField';
  */
 export default function InputNumber({data, className, style}) {
   const [cssClasses] = webexComponentClasses('adaptive-cards-input-number', className);
-  const [inputValue, setInputValue] = useState(data.value);
-  const handleInputChange = (value) => setInputValue(value);
+  const {setValue, getValue, setField} = useContext(AdaptiveCardContext);
+  const fieldState = getValue(data.id);
+
+  useEffect(() => {
+    setField({
+      id: data.id,
+      data: {
```

Comment:
no need for a substructure "data".  Just put everything on the first level (id, value, isRequired, etc.)